src/commands/radio/search.js

```diff
@@ -0,0 +1,54 @@
+const { SearchCommand, CommandError, SwitchbladeEmbed } = require('../../')
+
+module.exports = class TuneIn extends SearchCommand {
+  constructor (client) {
+    super(client, {
+      name: 'tunein',
+      category: 'radio',
+      parameters: [
+        {
+          type: 'string',
+          full: true
+        }
+      ],
+      requirements: { apis: ['tunein'] }
+    })
+  }
+
+  async search (_context, keyword) {
+    const results = await this.client.apis.tunein.search(keyword)
+    return results
+  }
```

Comment:
```js
  search (_context, keyword) {
    return this.client.apis.tunein.search(keyword)
  }
```

Reply:
HOW DID YOU CREATE A MULTILINE COMMENT? I spent 15 minutes yesterday trying to figure it out without any success >:c